_pages/software-tools/github.md

```diff
@@ -134,7 +134,7 @@ Contractors or external government collaborators should only be added to teams w
 
 ### Archiving
 
-As discussed in the [18F open source policy](https://github.com/18F/open-source-policy/blob/master/practice.md), we [archive](https://help.github.com/articles/archiving-repositories/) repositories to deprecate them. No approval is needed archive/unarchive a repository. Feel free to do so yourself, or ask [#admins-github](https://gsa-tts.slack.com/messages/admins-github) for help. Note that archiving a repository is _not_ the same as deleting it.
+As discussed in the [18F open source policy](https://github.com/18F/open-source-policy/blob/master/practice.md), we [archive](https://help.github.com/articles/archiving-repositories/) repositories to deprecate them. In short, that means we are no longer maintaining them, including keeping dependencies up-to-date. No approval is needed archive/unarchive a repository. Feel free to do so yourself, or ask [#admins-github](https://gsa-tts.slack.com/messages/admins-github) for help. Note that archiving a repository is _not_ the same as deleting it.
```

Comment:
```suggestion
As discussed in the [18F open source policy](https://github.com/18F/open-source-policy/blob/master/practice.md), we [archive](https://help.github.com/articles/archiving-repositories/) repositories to deprecate them. In short, that means we are no longer maintaining them, including keeping dependencies up-to-date. No approval is needed to archive/unarchive a repository. Feel free to do so yourself, or ask [#admins-github](https://gsa-tts.slack.com/messages/admins-github) for help. Note that archiving a repository is _not_ the same as deleting it.
```

Reply:
@afeld Is the advice about archiving/unarchiving whenever you want still correct?

Reply:
I think it's been working fine, yeah.

Reply:
To elaborate: Since anyone in @18F can create a new repository, that's effectively the same as the ability to unarchive, so I don't see any reason to put additional restrictions around the latter.

Reply:
I was wondering more about your team appearing to be the archivers/unarchivers, so just wanted to be sure. Thanks!